server/routes/security.js

```diff
@@ -57,7 +57,7 @@ const registerSchema = {
     phoneNumber: { type: "string" },
     address: { type: "string" },
     language: { type: "string" },
-    selectedSecurityQuestions: securityQuestionsSchema,
+    selectedSecurityQuestions: securityQuestionSchema,
```

Comment:
I have been using "securityQuestionsSchema" in my code. Are their many instances of "securityQuestionSchema"?